app/services/ocr_exporter.rb

```diff
@@ -36,9 +37,8 @@ def export
 
   def filenames(object)
     object.structural.contains.map do |file_set|
-      # TODO: using label in place of filename for now: https://github.com/sul-dlss/google-books/issues/444
-      file_set.structural.contains.find { |file| file.hasMimeType == 'text/plain' }&.label
-    end.compact
+      file_set.structural.contains.map { |file| file.filename.match?(/-gb-txt.zip/) ? file.filename : nil }
```

Comment:
You can simplify by using `filter` here rather than `map`

Reply:
Still need `map`, so I'd call it a toss-up on "simplify". Changed.